### ISSUE TITLE
A11y_AzureExtensionsforVSCode_AzureFuction_DefaultScreen_ColorContrast :The color contrast ratio of keyboard focus on several controls such as "Azure serverless community library" is less than 3:1.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-azurefunctions#2283]
**“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”**

GitHub Tags:#A11y_Azure Extensions for VS Code;#Azure Tools for VS Code;#A11yTCS;#A11yWCAG2.1;#MAC;#FTP;#Benchmark;#ColorContrast;#A11ySev2:#WCAG1.4.11;

### Environment Details:
Application Name: Azure Extensions for VS Code
Application Version: 1.48.0-insider
Commit: cfbd1999769f4f08dce29629fb92fdc0fac53829
Date: 2020-08-06T05:40:40.871Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina Version 10.15.6 (19G73)
Screen Reader: VoiceOver

### Repro Steps:
1. Open Visual studio code
2. Click on extension button and search for "Azure Functions"
3. Click on install and move to the "Azure" button, the "Default Screen" will appear.
4. Navigate to several controls such as "Azure serverless community library" using keyboard and verify if the color contrast ratio of keyboard focus is less than 3:1 or not.

### Actual:
The several controls such as "Azure serverless community library" has keyboard focus color contrast ratio of 1.5:1 which is less than the standard color contrast ratio of 3:1 which is not appropriate as the users with low vision will have difficulty in understanding the navigation on the screen.

### Expected:
The keyboard focus color contrast ratio of "Azure serverless community library" control should be greater than or equal to 3:1.

### User Impact:
If the color contrast ratio of the keyboard focus for the contents present on the screen is less than 3:1 then the users with low vision will have difficulty in understanding the keyboard navigation for the controls on the screen.

### Attachment link for Reference:
<img width="1403" alt="2283_A11y_AzureExtensionsforVSCode_AzureFuction_DefaultScreen_ColorContrast The color contrast ratio of keyboard focus on several controls such as  Azure serverless community library  is less than 31" src="https://user-images.githubusercontent.com/52036772/89622058-0a7c8d80-d8b0-11ea-80ea-7f8de1732289.png">

**Comment by EricJizbaMSFT** [comment]
This issue should be filed in the VS Code repo: https://github.com/Microsoft/vscode

Our extension has no control over the theming of links in markdown. For example, you can reproduce the same behavior without our extension. Create a "test.md" file, add markdown for a link, and open the markdown's preview.
![Screen Shot 2020-08-10 at 11 02 30 AM](https://user-images.githubusercontent.com/11282622/89815164-2888fe80-daf9-11ea-822e-5f77b23305cb.png)

**Comment by nirmalsahoo** [comment]
Logged bug in VS Code- https://github.com/microsoft/vscode/issues/104553
As per the C & AI bug process assigning the bug to the Product Owner. Please assign this bug back to the tester once the "External" issue has been resolved and changes are pushed to the environment that will be available for bug verification.

**Comment by EricJizbaMSFT** [comment]
Actually I think you selected the wrong colors. This is what I see:
![Screen Shot 2020-09-28 at 12 47 14 PM](https://user-images.githubusercontent.com/11282622/94478948-db381d80-0188-11eb-81e7-9bdbe500a860.png)
